packages/ui/src/components/file-toolbar-actions.tsx

```diff
@@ -16,25 +16,28 @@ export const FileToolbarActions: FC<FileToolbarActionsProps> = ({
   copyContent,
   showEdit = false
 }) => {
-  const commonButtonClasses = 'border-cn-borders-2 hover:bg-cds-background-4'
   return (
     <ButtonGroup spacing="0" className="h-full rounded">
```

Comment:
`ButtonGroup` will be a separate component, once it is added, this section will be refactored.